internal/config/cgmgr/cgmgr.go

```diff
@@ -79,8 +87,45 @@ func SetCgroupManager(cgroupManager string) (CgroupManager, error) {
 		}
 		return &systemdMgr, nil
 	case cgroupfsCgroupManager:
-		return new(CgroupfsManager), nil
+		return &CgroupfsManager{
+			memoryPath:    cgroupMemoryPathV1,
+			memoryMaxFile: cgroupMemoryMaxFileV1,
+		}, nil
```

Comment:
you may have missed it on the other thread, but @giuseppe corrected me, we can have cgroupv1 on cgroupfs. Can you add them back please?

Reply:
Yeah I think I missed that. I will add them back.
Thanks